### DISCUSSION TITLE
How to get the Galaxy Brain Achivement Github

### DISCUSSION THREAD

**Discussion by emohamedd** [community/community#46862] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

The "Galaxy Brain" achievement on GitHub is a reference to a meme related to having a large, complex and intelligent thought process. As far as I know, it is not an actual achievement that can be earned on the platform.

GitHub has a system of awards and achievements, but they are based on specific actions within the platform, such as making a certain number of contributions to repositories or creating a repository that becomes popular.

If you're interested in increasing your GitHub activity, some ways to do so include:

Contributing to open-source projects
Creating your own repository and contributing to it regularly
Participating in discussions and issues on repositories
Collaborating with others on projects
By consistently engaging with the GitHub community and contributing to the platform, you can establish yourself as an active and valuable member, which can help increase your visibility and recognition within the community.

**Comment by emohamedd** [comment]
I Hope i Answered The Question

**Reply by HassanOigag** [reply]
thanks A lot for ur explain

**Reply by emohamedd** [reply]
Ur Welcome

**Reply by RoshdiRaed** [reply]
thx

**Comment by SilasRodrigues19** [comment]
Hey @emohamedd 

Default `Galaxy Brain` achievement level, needs to have 2 discussions marked as answered.

The levels work like this:

- Default -> 2 accepted answers
- Bronze -> 8 accepted answers
- Silver -> 16 accepted answers
- Gold -> 32 accepted answers

There is a complete list of how to get the achievements in this link below:

https://github.com/drknzz/GitHub-Achievements

**Reply by emohamedd** [reply]
Good

**Reply by tsukiforge** [reply]
Good Bro Thank You

**Comment by YusufLisawi** [comment, marked as answer]
Default -> 2 accepted answers
    Bronze -> 8 accepted answers
    Silver -> 16 accepted answers
    Gold -> 32 accepted answers

**Reply by starlitnightsky** [reply]
Thank you!

**Comment by Hammythird** [comment]
That's great!!

**Comment by Faiqlatheef** [comment]
After answering two discussions, You will get Galaxy Brain badge
And Add 2 or more branches and create pull request and pull it You will get one
More details here:
https://github.com/Schweinepriester/github-profile-achievements

**Reply by starlitnightsky** [reply]
Thank you!

**Reply by tsukiforge** [reply]
Good

**Comment by HoseinRezaeeM** [comment]
How answer questijon?

**Comment by demirisdev** [comment]
Hi, can this be an answer?

**Reply by Blue-bird3** [reply]
and here

**Reply by kyuna0312** [reply]
yeah

**Comment by Blue-bird3** [comment]
ofc, it can

**Reply by demirisdev** [reply]
what?

**Reply by pwgit-create** [reply]
Your kindness was appreciated😀

**Comment by demirisdev** [comment]
what about this?

**Reply by Blue-bird3** [reply]
here it goes

**Reply by demirisdev** [reply]
that's nice too

**Reply by pwgit-create** [reply]
Indeed :)

**Comment by Blue-bird3** [comment]
that's nice

**Reply by demirisdev** [reply]
what's wrong

**Comment by github-staff** [comment]
Thanks so much for posting in our Community @emohamedd. Because of an influx of users creating low quality, nonsensical or spam posts in the Community with the aim of earning certain Achievements, we’ve made the difficult decision to disable the ability to earn Achievements in our Community. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Let us know if you have any other questions 🚀

**Comment by kyuna0312** [comment]
you should just answer any discussion then get badge xD 
is it can be a answer xD

**Comment by MoldyLunchBox** [comment]
i read somewhere that they removed support for this achievement . is it true?

**Comment by MahmudSiddekySwapnil** [comment]
thts great
